pkg/tfbridge/provider.go

```diff
@@ -815,9 +812,16 @@ func (p *Provider) Check(ctx context.Context, req *pulumirpc.CheckRequest) (*pul
 							Property: string(cf.Property),
 						})
 					} else {
-						logger.Warn(fmt.Sprintf("%v verification warning: %s: Examine values at %s", urn, e.Reason, e.ResourcePath))
+						logger.Warn(
+							fmt.Sprintf("Type checking failed:\n  Unexpected type at field %s: \n    %s", e.ResourcePath, e.Reason),
```

Comment:
It's a lot easier to tell that a field name is a field name if it is quoted.

```suggestion
							fmt.Sprintf("Type checking failed:\n  Unexpected type at field %q: \n    %s", e.ResourcePath, e.Reason),
```

Reply:
If there are multiple type failures, we will print
```
Type checking failed:
    Unexpected type at field...
Type checking failed:
    Unexpected type at field...
...
```

We should print the "Type checking failed:\n" (the prefix) just once, then each error message, then the suffix.

Reply:
Good call. I've updated this.